middle_end/flambda/types/inlining/inlining_arguments.mli

```diff
@@ -31,3 +31,19 @@ val meet : t -> t -> t
 val equal : t -> t -> bool
 
 val max_inlining_depth : t -> int
+
```

Comment:
The comment at the top of this file still isn't descriptive enough!

Reply:
Improved the comment.